module/zfs/vdev_label.c

```diff
@@ -1088,8 +1089,13 @@ vdev_uberblock_load_impl(zio_t *zio, vdev_t *vd, int flags,
 		vdev_uberblock_load_impl(zio, vd->vdev_child[c], flags, cbp);
 
 	if (vd->vdev_ops->vdev_op_leaf && vdev_readable(vd)) {
+		/*
+		 * If the pool was last imported by a non-MMP aware ZFS,
+		 * the "MMP BLOCKS" may contain valid uberblocks.
+		 */
+		int slots = VDEV_UBERBLOCK_COUNT(vd) + MMP_BLOCKS_PER_LABEL;
```

Comment:
We could, and I had the same thought, but after further reflection I'd like to leave it as is since it makes it clear that last uberblock spot is special.